runtime/frequency-rococo/src/lib.rs

```diff
@@ -0,0 +1,799 @@
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
is this just copied from frequency/src/lib.rs?  What's different?

Reply:
Runtime is copied as is for now, have added a chore #294 to refactor

Reply:
rococo runtime is now standalone and I would be include to use --features but will be a task in itself which can be tracked in above chore or converted to story